pipeline_manager/src/config.rs

```diff
@@ -110,6 +110,24 @@ pub(crate) struct ManagerConfig {
     #[serde(skip)]
     #[arg(short, long)]
     pub config_file: Option<String>,
+
+    /// [Developers only] Inject a SQL file into the database when starting the
+    /// manager.
+    ///
+    /// This is useful to populate the DB with state for testing.
+    #[serde(skip)]
+    #[arg(short, long)]
+    pub initial_sql: Option<String>,
```

Comment:
What's the use case for this? E.g., if you just want to add some projects to the DB, you could make some API calls after starting the server.

Reply:
Originally added it to test UI which didn't have create functionality. Figured it might be useful in the future too for patching or schema migration etc.
Yes we can do it with API calls too but reading a SQL file seemed like a way to do it easier/faster. But yes, it's not strictly necessary to have.